cpp/tests/CMakeLists.txt

```diff
@@ -6,9 +6,11 @@ project(CUDF_TESTS LANGUAGES C CXX CUDA)
 # - compiler function -----------------------------------------------------------------------------
 
 function(ConfigureTest CMAKE_TEST_NAME CMAKE_TEST_SRC)
+    message(STATUS "${TEST_NAME} will link against: gdf  ${ARROW_LIB}")
+
     add_executable(${CMAKE_TEST_NAME} ${CMAKE_TEST_SRC})
     set_target_properties(${CMAKE_TEST_NAME} PROPERTIES POSITION_INDEPENDENT_CODE ON)
-    target_link_libraries(${CMAKE_TEST_NAME} gmock gtest gmock_main gtest_main pthread cudf)
+    target_link_libraries(${CMAKE_TEST_NAME} gmock gtest gmock_main gtest_main pthread cudf rmm "${ARROW_LIB}")
```

Comment:
If `libarrow` is linked to `libcudf`, would it not also be linked to these tests? Ditto for `librmm` as it's linked to `cudf`.

Reply:
Understood and corrected